sequence_processing_pipeline/Pipeline.py

```diff
@@ -23,23 +25,26 @@ def __init__(self, configuration_file_path, input_directory, run_id):
         except JSONDecodeError:
             raise PipelineError(f'{configuration_file_path} is not a valid ',
                                 'json file')
-
         config = self.configuration['pipeline']
+        self.search_paths = config['search_paths']
+        self.run_id = run_id
+        # just take the first result for now
+        self.run_dir = self._search_for_run_dir(run_id)[0]
```

Comment:
This should always be 1 as the run_id can only exist in one of the options in the config file.

Reply:
Thanks! I'll change the method to always return the first one found.